server/player/player.go

```diff
@@ -874,14 +875,14 @@ func (p *Player) kill(src damage.Source) {
 		viewer.ViewEntityAction(p, entity.DeathAction{})
 	}
 
-	p.addHealth(-p.MaxHealth())
```

Comment:
Any reason for this change?

Reply:
yeah it was for checking if played has died or not in the drop items method